src/screens/ChamberOfSecrets.tsx

```diff
@@ -0,0 +1,222 @@
+/* eslint-disable react-native/no-raw-text */
+/* eslint-disable react-native/no-color-literals */
+/* eslint-disable react/no-unescaped-entities */
+import React, { useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { connect } from 'react-redux';
+
+import {
+  Button,
+  CheckBox,
+  EllipsisText,
+  InputItem,
+  LinearGradient,
+  RadioButton,
+  ScreenTemplate,
+  Text,
+} from 'app/components';
+import { HEADER_HEIGHT } from 'app/components/Header';
+import { ImportWalletType, Wallet } from 'app/consts';
+import { HDSegwitP2SHArWallet, HDSegwitP2SHAirWallet } from 'app/legacy';
+import { createPin, createTxPassword, setIsTcAccepted } from 'app/state/authentication/actions';
```

Comment:
you are using the same name when destructuring from props which can be confusing. I would do `createPin as createPinAction` and later in connect
export default connect(null, { createPin: createPinAction ... })(ChamberOfSecrets);

Reply:
To be honest, it isn't confusing for me at all but I believe you're right so I changed it as you advised.